mule-linter-core/src/main/groovy/com/avioconsulting/mule/linter/rule/configuration/MuleConfigSizeRule.groovy

```diff
@@ -7,19 +7,16 @@ import com.avioconsulting.mule.linter.model.rule.RuleViolation
 
 class MuleConfigSizeRule extends Rule {
 
-    static final String RULE_ID = 'MULE_CONFIG_SIZE'
+    static final String RULE_ID = 'MULE_CONFIG_FLOW_LIMIT'
```

Comment:
Generally, the rule id and rule class names are matching. Should we change the rule class name to be consistent?